tests/cdn/loadCKEditorCloud.test.ts

```diff
@@ -78,6 +81,72 @@ describe( 'loadCKEditorCloud', () => {
 		expect( loadedPlugins?.Plugin ).toBeDefined();
 	} );
 
+	describe( 'CSP', () => {
+		it( 'should set crossorigin=anonymous attribute on injected elements if `htmlAttributes` is not specified', async () => {
+			await loadCKEditorCloud( {
+				version: '43.0.0'
+			} );
+
+			expect( queryNonAnonymousLinks() ).toHaveLength( 0 );
+			expect( queryNonAnonymousScripts() ).toHaveLength( 0 );
+		} );
+
+		it( 'should set crossorigin=anonymous attribute on injected elements if `htmlAttributes` is specified but is blank', async () => {
+			await loadCKEditorCloud( {
+				version: '43.0.0',
+				injectedHtmlElementsAttributes: {}
+			} );
+
+			expect( queryNonAnonymousLinks() ).toHaveLength( 0 );
+			expect( queryNonAnonymousScripts() ).toHaveLength( 0 );
+		} );
+
+		it( 'should be possible to override the `crossorigin` attribute', async () => {
+			const promise = loadCKEditorCloud( {
+				version: '43.0.0',
+				injectedHtmlElementsAttributes: {
+					crossorigin: 'use-credentials'
+				}
+			} );
+
+			// It's fine because `use-credentials` throws an error in the vitest browser.
+			expect( promise ).rejects.toThrowError();
+		} );
+
+		it( 'should set nonce attribute on injected elements if `htmlAttributes` is specified', async () => {
+			await loadCKEditorCloud( {
+				version: '43.0.0',
+				injectedHtmlElementsAttributes: {
+					nonce: 'test-nonce'
+				}
+			} );
+
+			const script = createCKCdnUrl( 'ckeditor5', 'ckeditor5.umd.js', '43.0.0' );
+			const link = createCKCdnUrl( 'ckeditor5', 'ckeditor5.css', '43.0.0' );
+
+			expect( queryScript( script )?.getAttribute( 'nonce' ) ).toBe( 'test-nonce' );
+			expect( queryStylesheet( link )?.getAttribute( 'nonce' ) ).toBe( 'test-nonce' );
+
+			for ( const preloadLink of [ script, link ] ) {
+				expect( queryPreload( preloadLink )?.getAttribute( 'nonce' ) ).toBe( 'test-nonce' );
+			}
+		} );
+
+		function queryNonAnonymousScripts() {
+			return [ ...document.querySelectorAll( 'script' ) ].filter( script =>
+				script.getAttribute( 'src' )?.includes( CKBOX_CDN_URL ) &&
```

Comment:
```suggestion
				script.getAttribute( 'src' )?.includes( CK_CDN_URL ) &&
```

This function always returns 0 matches.

Reply:
Yep, good catch!